### ISSUE TITLE
Parent transform effect influence does not work

### ISSUE THREAD

**Issue by samytichadou** [friction2d/friction#123]
Friction 0.9.5 on kde neon 5.27
When parenting objects to a null with a parent transform effect, the influence (loc, rot, scale) does not affect the behavior.
This could be a great fix to create simple parallax effect through a null controler.
Thanks for your work, cheers !

**Comment by rodlie** [comment]
Never noticed this issue before, and it's almost a year old 😄

> When parenting objects to a null with a parent transform effect, the influence (loc, rot, scale) does not affect the behavior.

Could you give me something to replicate (a step-by-step) so I don't missunderstand anything :)

**Comment by samytichadou** [comment]
> Never noticed this issue report, and it's almost a year old 😄

np, hard to keep track of everything !

### To reproduce
- Parent a layer to another with Parent effect
![Image](https://github.com/user-attachments/assets/66e1f2af-b2d1-4bae-950e-0d55159c73a4)

- Move parent layer (loc rot scale)

- Turn down the Parent effect influence, nothing happened
![Image](https://github.com/user-attachments/assets/01146ae7-4977-498d-b63f-f07189d24494)

### Expected behavior
The influences when turned down should makes the child unresponsive to the parent's transformations.
He should go back to its initial loc/rot/scale.

**Comment by rodlie** [comment]
Thanks for the info, will look into the issue later today or tomorrow.

**Comment by rodlie** [comment]
> hard to keep track of everything !

Yeah, I still got 60+ unread emails related to Friction. I just don't have time to follow up on everything. People will need to ping me on important issues I forget.
